### ISSUE TITLE
Build failure: deno-2.1.4

### ISSUE THREAD

**Issue by khaneliman** [NixOS/nixpkgs#367962]
## Steps To Reproduce

1. build deno-2.1.4

## Build log

<details>

<summary>Build Log</summary>

```
libtool: compile:  /nix/store/wkadfwbhn9b4kdih88j7izzy20qw79bk-clang-wrapper-19.1.5/bin/cc -DHAVE_CONFIG_H -I. -I.. -I. -I../include -Iinclude -I../src -I. -I../include -Iinclude -I../src -Oz -ffunction-sect>
  make[2]: Leaving directory '/private/tmp/nix-build-deno-2.1.4.drv-1/source/target/aarch64-apple-darwin/release/build/libffi-sys-d47a2cef7411a0b3/out/libffi-build/aarch64-apple-darwin'
  make[1]: Leaving directory '/private/tmp/nix-build-deno-2.1.4.drv-1/source/target/aarch64-apple-darwin/release/build/libffi-sys-d47a2cef7411a0b3/out/libffi-build/aarch64-apple-darwin'

  --- stderr
  clang: error: unsupported option '-print-multi-os-directory'
  clang: error: no input files
  ../src/java_raw_api.c:328:46: warning: 'ffi_java_raw_size' is deprecated [-Wdeprecated-declarations]
    328 |   ffi_java_raw *raw = (ffi_java_raw*)alloca (ffi_java_raw_size (cif));
        |                                              ^
  include/ffi.h:326:56: note: 'ffi_java_raw_size' has been explicitly marked deprecated here
    326 | size_t ffi_java_raw_size (ffi_cif *cif) __attribute__((deprecated));
        |                                                        ^
  ../src/java_raw_api.c:331:3: warning: 'ffi_java_ptrarray_to_raw' is deprecated [-Wdeprecated-declarations]
    331 |   ffi_java_ptrarray_to_raw (cif, avalue, raw);
        |   ^
  include/ffi.h:322:93: note: 'ffi_java_ptrarray_to_raw' has been explicitly marked deprecated here
    322 | void ffi_java_ptrarray_to_raw (ffi_cif *cif, void **args, ffi_java_raw *raw) __attribute__((deprecated));
        |                                                                                             ^
  2 warnings generated.
  /private/tmp/nix-build-deno-2.1.4.drv-1/sysv-98dd48.s:27:2: error: invalid CFI advance_loc expression
   .cfi_def_cfa x1, 40;
   ^
  /private/tmp/nix-build-deno-2.1.4.drv-1/sysv-98dd48.s:188:2: error: invalid CFI advance_loc expression
   .cfi_adjust_cfa_offset (8*2 + (8 * 16 + 8 * 8) + 64)
   ^
  make[2]: *** [Makefile:1296: src/aarch64/sysv.lo] Error 1
  make[1]: *** [Makefile:1397: install-recursive] Error 1
  make: *** [Makefile:3158: install] Error 2
  thread 'main' panicked at /private/tmp/nix-build-deno-2.1.4.drv-1/deno-2.1.4-vendor.tar.gz/libffi-sys/build/common.rs:8:5:
```

</details>

## Metadata

- system: `"aarch64-darwin"`
 - host os: `Darwin 24.2.0, macOS 15.2`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.25.3`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/store/73cnf2k7mji95fcflvay15pdw4skhhk5-source`
 
## Notify maintainers
@jk

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by ofalvai** [comment]
I have a fix in process at #367962

**Comment by 06kellyjac** [comment]
Thanks for raising @khaneliman
Just for future reference maintainer names within the nix codebase don't relate 1-to-1 with github so I wasn't aware of this issue.

To get a maintainer's GH @ you can go via search.nixos.org

![image](https://github.com/user-attachments/assets/80d61129-e0b2-4046-b34f-d58d3102a77a)

Or if you have nixpkgs open in a terminal you can use `./maintainers/scripts/get-maintainer.sh`

Cheers
